daemon/cmd/daemon_main.go

```diff
@@ -288,10 +288,11 @@ func initializeFlags() {
 	flags.StringSlice(option.IPv6PodSubnets, []string{}, "List of IPv6 pod subnets to preconfigure for encryption")
 	option.BindEnv(Vp, option.IPv6PodSubnets)
 
-	flags.Var(option.NewNamedMapOptions(option.IPAMMultiPoolNodePreAlloc, &option.Config.IPAMMultiPoolNodePreAlloc, nil),
-		option.IPAMMultiPoolNodePreAlloc, "List of IP pools which should be pre-allocated on this node")
-	flags.MarkHidden(option.IPAMMultiPoolNodePreAlloc)
-	option.BindEnv(Vp, option.IPAMMultiPoolNodePreAlloc)
+	flags.Var(option.NewNamedMapOptions(option.IPAMMultiPoolPreAllocation, &option.Config.IPAMMultiPoolPreAllocation, nil),
+		option.IPAMMultiPoolPreAllocation,
+		fmt.Sprintf("Defines how the minimum number of IPs a node should pre-allocate from each pool (default %s)", defaults.IPAMMultiPoolPreAllocation))
```

Comment:
I am torn on this. It's not really a string, it's a map. But `option.NewNamedMapOptions` does not allow me to define a default map. I've went back and forth between `%s` and `%q`. I ended up with `%s`, because other non-string types (like e.g. integers, durations and booleans) don't use quotes either: https://github.com/cilium/cilium/blob/d137d831a26ba72fa85db4972f32737d28068a9e/Documentation/cmdref/cilium-agent.md?plain=1#L17

Reply:
Another note here is that you don't need to explicitly mention the default here because `flags` does it for you, AFAIR.

Reply:
Unfortunately, for some reason it doesn't here. See the generated docs, there is no duplicate `default`.